api/Worklet.json

```diff
@@ -66,6 +66,45 @@
             "deprecated": false
           }
         }
+      },
+      "ecmascript_modules": {
+        "__compat": {
+          "description": "Support for ECMAScript modules",
+          "support": {
+            "chrome": {
+              "version_added": "80"
```

Comment:
I am not certain of this - can't find info either way - but is likely. The wpt tests only run locally so can't test.